package-lock.json

```diff
@@ -0,0 +1,6158 @@
+{
+  "name": "yext-js",
```

Comment:
This file shouldn't be added. Same in pages-components. This library uses pnpm, not npm.

Reply:
Still here